shell/platform/tizen/flutter_tizen_view.cc

```diff
@@ -73,12 +73,13 @@ void FlutterTizenView::SetEngine(std::unique_ptr<FlutterTizenEngine> engine) {
   // Set up window dependent channels.
   BinaryMessenger* messenger = internal_plugin_registrar_->messenger();
 
+  platform_channel_ =
+      std::make_unique<PlatformChannel>(messenger, tizen_view_.get());
```

Comment:
nit: This might be moved to line 85. (No blank line between platform_channel and text_input_channel.)